extensions/wikia/FounderEmails/events/FounderEmailsViewsDigestEvent.class.php

```diff
@@ -27,11 +27,16 @@ public function process ( Array $events ) {
 
 		$wgTitle = Title::newMainPage();
 		// Get list of founders with digest mode turned on
-		$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-views-digest' );
+		if ( empty( $events ) ) {
+			$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		} else {
+			$cityList = $events;
```

Comment:
The comment for this method says that `$events` should always be empty when this is called, why are we setting cityList to it?